aiida/tools/ipython/ipython_magics.py

```diff
@@ -81,8 +81,13 @@ def aiida(self, line='', local_ns=None):
         from aiida.cmdline.utils.shell import get_start_namespace
 
         self.is_warning = False
-        load_profile()
-        self.current_state = 'Loaded AiiDA DB environment.'
+        lcontent = line.strip()
+        if lcontent:
```

Comment:
Does it make sense to do a try/except instead, if the provided profile name cannot be loaded?

Reply:
At the moment it will raise an exception, which I think is verbose enough? 

If I do `%aiida bla`, then it gives:

```
    234         if name not in self.profile_names:
--> 235             raise exceptions.ProfileConfigurationError('profile `{}` does not exist'.format(name))
    236
    237     def get_profile(self, name=None):

ProfileConfigurationError: profile `bla` does not exist
```

Reply:
Seems fine to me